tests/test__collection_api.py

```diff
@@ -2065,6 +2065,30 @@ def test__aggregate_sample_errors(self):
             with self.assertRaises(mongomock.OperationFailure):
                 self.db.a.aggregate([{'$sample': case}])
 
+    def test__aggregate_count(self):
+        self.db.a.insert_many([
```

Comment:
Please add the equivalent test in test__mongomock as well.

Reply:
I just added https://github.com/mongomock/mongomock/pull/479/commits/841f6c2523407721bc5c18c7f164e73baaccbceb. Not sure if this is enough for you